great_expectations/data_context/data_context.py

```diff
@@ -2850,7 +2850,7 @@ def profile_data_asset(
         )
         profiling_results["results"].append((expectation_suite, validation_results))
 
-        self.validations_store.set(
+        validation_ref = self.validations_store.set(
```

Comment:
I think we ultimately want to add a specific general `GeCloudIdentifier` that's used for all object types if GE Cloud is in use, which will allow us to get rid of `from_ge_tuple` and just have a specific `from_tuple` that's used for all GE Cloud things. 

I think this is fine for now. I have a bunch of stuff done already for an updated DataContext with a `ge_mode` flag - once I get back to it, I can take care of creating `GeCloudIdentifier` so that the abstractions are much cleaner.